src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/controller/admin/LocationGroupAdminController.kt

```diff
@@ -144,7 +146,7 @@ class LocationGroupAdminController(
   }
 
   @PreAuthorize("hasRole('VISIT_SCHEDULER_CONFIG')")
-  @DeleteMapping(REFERENCE_LOCATION_GROUP_ADMIN_PATH)
+  @DeleteMapping(REFERENCE_LOCATION_GROUP_ADMIN_PATH, produces = [MediaType.APPLICATION_JSON_VALUE])
```

Comment:
don't normally use this, but I guess I normally return a DTO so it knows it's JSON